src/Core/Other.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @param {(Component|Element|DOM|Node)} element
+ * @return {Node?}
+ */
+function findDOMNode (element) {
+	if (element) {
+		if (isValidElement(element[SymbolElement]))
+			return findDOMNode(element[SymbolElement])
+
+		if (isValidElement(element)) {
+			if (element.flag < ElementPortal)
+				return findDOMNode(elementSibling(element, 1))
+			else if (element.DOM)
```

Comment:
Maybe this should warn/throw if this returns false, since that can only mean you're trying to call this on an unmounted component.

Reply:
seems reasonable